From 55be405630d4e16215d349bb9ce45c7165fc382d Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 26 Jan 2024 18:02:20 +0100
Subject: [PATCH 0584/1695] dts: stm32: VDDGPU no more always-on for
 stm32mp257f-dk

The GPU power domain shall be powered on only when
the IP is in use. This is to avoid a permanent leakage
current.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 9d69054b01418f130da9650b3d0481a11a38e4cf dts: stm32: VDDGPU no more always-on for stm32mp257f-dk]
Signed-off-by: Olivier BIDEAU <olivier.bideau@st.com>
Change-Id: I1a141caabeefd427eb024e56db28d03276a26b35
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/347183
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Tested-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-dk.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index c709304df..b87d0556c 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -116,7 +116,6 @@
 				regulator-name = "vddgpu_pmic";
 				regulator-min-microvolt = <800000>;
 				regulator-max-microvolt = <900000>;
-				regulator-always-on;
 			};
 			vddio_pmic: buck4 {
 				regulator-name = "vddio_pmic";
-- 
2.47.2

